local/DirLocations.ini

```diff
@@ -0,0 +1,12 @@
+; A parent local directory can be specified. When resolving paths to configuration files
+; this dir will also be searched if the file does not exist in the current dir.
+[Parent_Dir]
+; Path to the parent directory
+;path =
+; Whether the path is relative or absolute (defaults to false)
```

Comment:
I went ahead and added a note to the comment here to indicate the default value, since it's useful to know for reference.

Reply:
Thank you. This definitely makes sense.